package.json

```diff
@@ -15,7 +15,10 @@
     "@semantic-release/github": "^7.0.7",
     "@semantic-release/npm": "^7.0.5",
     "@semantic-release/release-notes-generator": "^9.0.1",
-    "semantic-release": "^17.1.1"
+    "execa": "^4.0.3",
+    "file-url": "^3.0.0",
+    "semantic-release": "^17.1.1",
+    "tempy": "^0.6.0"
```

Comment:
1) These look more like `devDependencies`?
2) While `execa` does provide some nice sugar atop native Node.js API, `tempy` can be fairly easily replaced with `os.tmpdir()`?

Reply:
1. fixed with f4d5059
2. I like `tempy` because it ensures already that there are no collisions on the generated directories. If I we can replace it but I don't have time now for it :) I don't really see it as a major issue, but I can happily take upon it next time I am working on maintenance.